lib/rules/shorthand-property-no-redundant-values/index.js

```diff
@@ -15,7 +14,14 @@ const messages = ruleMessages(ruleName, {
   rejected: (unexpected, expected) => `Unexpected longhand value '${unexpected}' instead of '${expected}'`,
 })
 
-const shorthandableProperties = new Set(Object.keys(shorthandData))
+const shorthandableProperties = new Set([
```

Comment:
Done.  Not sure if text correct, though. Also added `grid-gap`. Maybe some other Grid properties can be added, but I can't say for sure.